core/shared/src/main/scala/zio/ZManaged.scala

```diff
@@ -1539,12 +1543,26 @@ object ZManaged {
    * Creates a scope in which resources can be safely preallocated.
    */
   def scope[R]: ZManaged[R, Nothing, Scope] =
```

Comment:
How about we rename `scope` to `preallocationScope`, and give the `scope` names to the new combinator?

Reply:
This way I wouldn't be entirely lying here: https://twitter.com/iravid_/status/1180569798037708800?s=20 😇

Reply:
Nice, will update 😁